app/javascript/src/dashboard.js

```diff
@@ -116,24 +71,141 @@ $('document').ready(() => {
     }
   )
 
+  const handleAjaxError = e => {
+    if (e.status === 401) {
+      location.reload();
+    } else {
+      console.log(e);
+      if (e.responseJSON && e.responseJSON.error) {
+        alert(e.responseJSON.error);
+      } else if (e.response.statusText) {
+        alert(`Unable to complete your request: ${e.response.statusText}`);
```

Comment:
We will want to make this message more user-friendly - something like "Sorry, try that again?" and "Sorry, if you're seeing a problem, please fill out the Report A Site Issue 
link to the bottom left near your email address" (is that too long?)
http://meetcontent.com/blog/the-little-things-why-microcopy-matters/